webpack/assets/javascripts/react_app/components/common/table/actions/getTableItemsAction.js

```diff
@@ -6,10 +6,11 @@ import createTableActionTypes from '../actionsHelpers/actionTypeCreator';
  * An async Redux action that fetches and stores table data in Redux.
  * @param  {String} controller the controller name
  * @param  {Object} query      the API request query
+ * @param  {String} plugin     the foreman plugin name
```

Comment:
there are some plugins that don't use their name for the API url (i.e  in openscap the api url is `/api/compliance/<controller>`)